src/test/kotlin/uk/gov/justice/hmpps/offenderevents/resource/QueueListenerIntegrationTest.kt

```diff
@@ -2,64 +2,49 @@ package uk.gov.justice.hmpps.offenderevents.resource
 
 import com.amazonaws.services.sqs.AmazonSQS
 import com.amazonaws.services.sqs.model.PurgeQueueRequest
-import org.awaitility.Awaitility
-import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.beans.factory.annotation.Qualifier
+import org.awaitility.kotlin.await
 import org.springframework.boot.test.mock.mockito.SpyBean
-import uk.gov.justice.hmpps.offenderevents.config.SqsConfigProperties
-import uk.gov.justice.hmpps.offenderevents.config.prisonEventQueue
-import uk.gov.justice.hmpps.offenderevents.subscribe.hmppsEventTestQueue
-import uk.gov.justice.hmpps.offenderevents.subscribe.prisonEventTestQueue
+import uk.gov.justice.hmpps.sqs.HmppsQueue
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
 
 abstract class QueueListenerIntegrationTest : IntegrationTestBase() {
 
   @SpyBean
-  @Qualifier("awsSqsClient")
-  protected lateinit var awsSqsClient: AmazonSQS
+  lateinit var hmppsQueueService: HmppsQueueService
```

Comment:
Not sure this needs to be a spybean?

Reply:
nope - left over from previous file - updated